include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -168,25 +176,30 @@ constexpr char METRIC_TYPE_TIMING[] = "ms";
 constexpr char METRIC_TYPE_SET[] = "s";
 }  // namespace detail
 
-inline StatsdClient::StatsdClient(const std::string& host,
+inline StatsdClient::StatsdClient(const std::string &host,
                                   const uint16_t port,
-                                  const std::string& prefix,
+                                  const std::string &prefix,
                                   const uint64_t batchsize,
-                                  const uint64_t sendInterval) noexcept
-    : m_prefix(detail::sanitizePrefix(prefix)), m_sender(new UDPSender{host, port, batchsize, sendInterval}) {
+                                  const uint64_t sendInterval,
+                                  const unsigned int gaugePrecision) noexcept
+    : m_prefix(detail::sanitizePrefix(prefix)),
+      m_sender(new UDPSender{ host, port, batchsize, sendInterval }) {
     // Initialize the random generator to be used for sampling
     seed();
     // Avoid re-allocations by reserving a generous buffer
     m_buffer.reserve(256);
+    this->gaugePrecision = gaugePrecision;
```

Comment:
a couple notes on style:

it seems that your formatter is doing some strange things with the white space which is makign the diff hard to read. we are using clangformat here. also notice that we use `hungarian` notation, so that variable should be `m_guagePrecision` since its a member variable. also, i would suggest not using `this->` its superfluous (and looks like java :trollface: )

Reply:
- Ran clang-format again.
- Changed instance variable name.